services/state/contract/bytecode.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.contract">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * The bytecode for a contract id, along with its hash. (Computing this hash
+ * is fairly expensive, and may be in principle be requested frequently within
+ * an EVM transaction.)
+ */
```

Comment:
Yes, expensive to compute, but is there a reason it is part of the message?  If expensive to compute can't the message receiver calculate it and keep it?  Or is it that an _instance_ of this message is _consumed_ multiple times   and so it's cheaper for the sender to compute it once, rather than each receiver once?

Reply:
Removed this field.